src/Trakx.Kaiko.ApiClient.Tests/Integration/GenerateApiClientChecker.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Text.RegularExpressions;
+
+namespace Trakx.Kaiko.ApiClient.Tests;
+
+public class GenerateApiClientChecker
+{
+    public ITestOutputHelper Output { get; }
+
+    public GenerateApiClientChecker(ITestOutputHelper output)
+    {
+        Output = output;
+    }
+
+    [Fact]
```

Comment:
❤️ thanks!! can go to Common.Testing soon ? 😉

Reply:
Cool, I'll ask @joaofbvieira to include it as part of his refactor.